doc/user/content/sql/system-catalog/mz_catalog.md

```diff
@@ -273,7 +273,7 @@ Field        | Type     | Meaning
 -------------|----------|--------
 `id`         | [`text`] | The ID of the list type.
 `element_id` | [`text`] | The IID of the list's element type.
-`element_modifiers` | [`int8 list`] | The element type modifiers, or `NULL` if none.
+`element_modifiers` | [`uint8 list`] | The element type modifiers, or `NULL` if none.
```

Comment:
The link was broken, and I couldn't find any other instance of `int8` in the docs. On another look, we use `bigint` consistently instead of `int8` (they're [aliases](https://materialize.com/docs/sql/types/integer/#bigint-info)), so reverted this back as `bigint list`.